Frontend/src/views/WalksDetail.vue

```diff
@@ -8,6 +8,14 @@
             <PhotowalkMap/>
           </v-col>
           <v-col cols="12" md="6">
+              <h3 class="text-h3" v-text="photowalkTitle"/>
+              <h4 class="text-h4" v-text="photowalkRegion"/>
+              <span v-text="photowalkLength"/>
+            <v-row>
+              <p v-if="this.$store.state.detail.currentID === 1">{{$t('photos.description1')}}</p>
+              <p v-if="this.$store.state.detail.currentID === 2">{{$t('photos.description2')}}</p>
+              <p v-if="this.$store.state.detail.currentID === 3">{{$t('photos.description3')}}</p>
```

Comment:
Lässt sich abkürzen: 
<p>{{$t('photos.description'+ this.$store.state.detail.currentID)}}</p>